projects/angular/src/wizard/STYLES.md

```diff
@@ -8,11 +8,14 @@
 | --clr-wizard-stepnav-bgcolor                   | Wizard navigation background color                                                        |
 | --clr-wizard-main-text-color                   | Wizard text color                                                                         |
 | --clr-wizard-stepnav-text                      | Wizard navigation text color                                                              |
-| --clr-wizard-stepnav-text--active              | Wizard navigation text color of the currently active element                              |
+| --clr-wizard-stepnav-text-active               | Wizard navigation item color when active                                                  |
+| --clr-wizard-stepnav-text-hover                | Wizard navigation item color when hovered                                                 |
+| --clr-wizard-stepnav-text--active              | Wizard navigation item color when selected (misnamed)                                     |
+| --clr-wizard-stepnav-text-disabled             | Wizard navigation item color when disabled                                                |
 | --clr-wizard-title-text                        | Wizard title text color                                                                   |
 | --clr-wizard-stepnav-border-size               | Wizard navigation border size                                                             |
 | --clr-wizard-stepnav-item-border-color         | Wizard navigation border color                                                            |
-| --clr-wizard-stepnav-item-border-color--active | Wizard navigation border color for the currently active element                           |
+| --clr-wizard-stepnav-item-border-color--active | Wizard navigation item complete border color (misnamed)                                   |
```

Comment:
We can add deprecation for --active properties in favor of selected and add thme for removal in v18.
```suggestion
| --clr-wizard-stepnav-item-border-color--active | Deprecated in v17 in favor of  `--clr-wizard-stepnav-item-border-color-selected`                                    |
| --clr-wizard-stepnav-item-border-color-selected | Wizard navigation item complete border color                                   |
```